components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/SingleComponentConfig.scala

```diff
@@ -21,6 +21,29 @@ import pl.touk.nussknacker.engine.api.definition.{ParameterEditor, ParameterVali
   def paramConfig(name: String): ParameterConfig = params.flatMap(_.get(name)).getOrElse(ParameterConfig.empty)
 }
 
+@JsonCodec case class SingleComponentUIConfig(
```

Comment:
It isn't serialized to json am I right? I'd rather move this class next to `AdditionalComponentsUIConfigProvider` and treat it as a part of this interface. Also let's rename it to something better, "UI" looks like it is a DTO returned to FE. Mabye `ComponentConfigWithoutId`? or `BaseComponentConfig`

Reply:
1. Please do not resolve reviewer's comments - I want to check of my own if this comment is to resolve
2. I also wrote about "I'd rather move this class next to AdditionalComponentsUIConfigProvider" and you didn't do that.